datalake/src/test/java/com/sequenceiq/datalake/service/upgrade/recovery/SdxUpgradeRecoveryServiceTest.java

```diff
@@ -65,7 +64,6 @@ public void setup() {
         request = new SdxRecoveryRequest();
         request.setType(SdxRecoveryType.RECOVER_WITHOUT_DATA);
         when(cluster.getClusterName()).thenReturn(CLUSTER_NAME);
-        when(sdxService.getByNameOrCrn(USER_CRN, NameOrCrn.ofName(CLUSTER_NAME))).thenReturn(cluster);
```

Comment:
pls remove this, too, as it is not used anymore here:

    @Mock
    private SdxService sdxService;